static/app/gettingStartedDocs/javascript/angular.tsx

```diff
@@ -74,44 +49,45 @@ const getNextStep = (
   return nextStepDocs;
 };
 
-function getNpmPackage(angularVersion) {
-  return angularVersion === AngularVersion.V12
-    ? '@sentry/angular-ivy'
-    : '@sentry/angular';
-}
-
-const getInstallConfig = (params: Params) => [
+const getInstallConfig = () => [
   {
     language: 'bash',
     code: [
       {
         label: 'npm',
         value: 'npm',
         language: 'bash',
-        code: `npm install --save ${getNpmPackage(params.platformOptions.siblingOption)}`,
+        code: `npm install --save @sentry/angular`,
       },
       {
         label: 'yarn',
         value: 'yarn',
         language: 'bash',
-        code: `yarn add ${getNpmPackage(params.platformOptions.siblingOption)}`,
+        code: `yarn add @sentry/angular`,
+      },
+      {
+        label: 'pnpm',
+        value: 'pnpm',
+        language: 'bash',
+        code: `pnpm install @sentry/angular`,
       },
     ],
   },
 ];
 
-const onboarding: OnboardingConfig<PlatformOptions> = {
-  install: (params: Params) => [
+const onboarding: OnboardingConfig = {
+  install: () => [
     {
       type: StepType.INSTALL,
       description: tct(
-        'Add the Sentry SDK as a dependency using [codeNpm:npm] or [codeYarn:yarn]:',
+        'Add the Sentry SDK as a dependency using [codeNpm:npm], [codeYarn:yarn] or [codePnpm:pnpm]:',
```

Comment:
Should we add `pnpm` everywhere?

Reply:
let's look at this later, but possibly!

Reply:
+1 to tackle this in a follow-up PR but I'd vote yes, generally